scripts/apps/authoring/views/item-carousel.html

```diff
@@ -11,132 +11,132 @@
         <a ng-click="navNext()" class="sd-media-carousel__nav-button sd-media-carousel__nav-button--next-inner"></a>
 
         <div class="sd-media-carousel__content owl-carousel">
-            <div class="sd-media-carousel__page" ng-repeat="item in carouselItems">
+            <div class="sd-media-carousel__page" ng-repeat="carouselItem in carouselItems">
```

Comment:
Why did you need to rename this?

Reply:
Whenever we click on `edit image or edit metadata` etc. buttons, we are calling a method like `associations.edit(this, carouselItem[carouselItem.fieldId])`, where `this` is the directive scope.
In the directive scope `scope.item` is the one which contains the media-gallery, slugline, body, etc. But when we do `ng-repeat="item in carouselItems"`, this ng-repeat `item` overwrites the directive scope item. Which means we don't get the expected item on doing `associations.edit(this, carouselItem[carouselItem.fieldId])`. Which causes the errors.